tests/Unit/Models/ProjectTest.php

```
@@ -102,6 +103,44 @@ public function findByOrgIdReturnsEmptyArrayWhenNone(): void
     }
 
+    #[Test]
+    public function findActiveSummariesByOrgIdExcludesDeletedProjectsAndOrdersByName(): void
+    {
+        $capturedSql = null;
+        $capturedParams = null;
+        $stmt = $this->createMock(\PDOStatement::class);
+        $stmt->method('fetchAll')->willReturn([
+            ['id' => 33, 'name' => 'Alpha Project'],
+        ]);
+        $db = $this->createMock(Database::class);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)